lib/streamlit/web/cli.py

```diff
@@ -14,10 +14,13 @@
 
 """A script which is run when the Streamlit package is executed."""
 
+from __future__ import annotations
+
 import os
 import sys
-from typing import Any, Dict, List, Optional
+from typing import Any, Dict, List
 
+# We cannot lazy-load click here because its used via decorators.
 import click
```

Comment:
Hm, do we get any performance benefit from lazy loading `click` in this case given that this is module is loaded pretty close to the beginning of streamlit starting?

(I guess it's possible that we don't see any gain in open source but may for code creating its own Streamlit Runtime?)

Reply:
Yep, `click` is a bit special on this aspect compared to the other lazy-loaded modules since we won't have any advantage in normal Streamlit usage. But I think it would allow SiS to ignore `click` as a dependency (-> faster installation time), and it allows to ignore it for `stlite` as well.  In the `stlite` case, the downloading time of external packages is the biggest chunk of the overall loading time and the loading times is one of the bigger hurdles for using it in our docs & as playground.